### ISSUE TITLE
Fix the unintended warning from parallel computing

### ISSUE THREAD

**Issue by iok-ui** [softmatterlab/BRAPH-2-Matlab#1569]
**Describe the bug**
A warning about parallel computing while calculating COMPARISON ENSEMBLE.

The warning content is the following:
```
Warning: During load:
 An invalid default object has been detected while loading a heterogeneous array of class event.listener. An empty array of class event.listener will be returned.
> In parallel.internal.pool.deserialize (line 33)
In parallel.internal.pool.deserializeFunction (line 17)
In remoteParallelFunction (line 29)
```

**To Reproduce**
_Cannot to be reproduced_ but it happens by doing the following steps:
1. Make sure the parallel pool are running (check parallel status indicator in the lower left corner of the MATLAB desktop)  
2. Open pipeline "Pipeline Connectivity Comparsion BUT"
3. Go to the last step "5. Comparison"
4. Open the CompareEnsemble GUI, go to Group-averaged MEASURES, and calculate Degree
5. See error

**Expected behavior**
The warning will repeat itself several times until the completion of the calculation.

**Screenshots**
![image](https://github.com/giovannivolpe/BRAPH-2-Matlab-beta/assets/30530538/98b8d06e-1f63-476e-9752-cc6f64bd480d)
<img width="440" alt="image" src="https://github.com/braph-software/BRAPH-2/assets/30530538/c9e63733-47ce-45cf-9bec-b0162cbd5a70">

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version R2022b (Windows), R2022a (Linux)

**Additional context**
This doesnt happen in AnalyzeEnsemble.

**Comment by egolol** [comment]
@c-yuwei can you confirm that this behaviour keeps occurring to you? I tested this and it does not happen anymore.

**Comment by iok-ui** [comment]
@egolol No, I cannot replicate this error. But evidently, it happens to Jiawei (on Oct 18), Anna, Hang, and others. Not sure if they can reproduce it.
